src/Api.hs

```diff
@@ -36,8 +41,21 @@ data SearchResult = SearchResult
 
 instance ToJSON SearchResult
 
-type ParaApi
-   = "search" :> ReqBody '[ JSON] Tags :> Post '[ JSON] SearchResults :<|> "email" :> Capture "EMAIL" String :> Get '[ JSON] Tags :<|> "save" :> Capture "email" String :> ReqBody '[ JSON] Tags :> Post '[ PlainText] String :<|> Raw
+instance ElmType SearchResult
+
+newtype Saved = Saved
+  { saved :: String
+  } deriving (Generic)
+
+instance ToJSON Saved
+
+instance ElmType Saved
+
+type ParaApi = "search" :> ReqBody '[ JSON] Tags :> Post '[ JSON] SearchResults
```

Comment:
I split the api, because generating does not like `Raw`, so we just generate the elm stuff for the used endpoints